lib/PuppeteerSharp.Tests/PuppeteerTests/PuppeteerLaunchTests.cs

```diff
@@ -156,15 +156,15 @@ public async Task UserDataDirOptionShouldRestoreCookies()
                 {
                     var page = await browser.NewPageAsync();
                     await page.GoToAsync(TestConstants.EmptyPage);
-                    await page.EvaluateExpressionAsync(
+                    await page.EvaluateExpressionAsync<object>(
```

Comment:
can this use the JObject overload without specifying `<object>`?

Reply:
As I mentioned in the PR description, I thought by removing the default object and dynamic methods we would make users think about what they want, I can add back methods defaulting to object though ?